web/packages/shared/components/UnifiedResources/types.ts

```diff
@@ -115,15 +116,17 @@ export type UnifiedResourceUi = {
   ActionButton: React.ReactElement;
 };
 
+export type UnifiedResourceDefinition =
```

Comment:
This doesn't seem to be used anywhere else, so I think it'd be better to not export it. Also, does it need to have "Definition" in its name? It feels a bit redundant, it's like calling a type "FooType" instead of "Foo" or a list of names "namesList" instead of "names".

Reply:
this is also being used in another PR, but i'll remove it from this PR. I'll rename it. I thought UnifiedResource is taken, but it's in another file (service/resource.ts)